go-controller/pkg/ovn/policy.go

```diff
@@ -3,9 +3,12 @@ package ovn
 import (
 	"fmt"
 	"net"
+<<<<<<< HEAD
```

Comment:
looks like a mistake in this commit

Reply:
Sorry I just need to squash those two commits the end state is correct.